README.md

```
@@ -1,63 +1,27 @@
-
-<!-- TODO -->
+TODO
 
 ## Contribution & Attribution
 
 ### Project Contributors
 
-<a href="https://github.com/toddeTV/printed-europe.com/graphs/contributors">
-<img src="https://contrib.rocks/image?repo=toddeTV/printed-europe.com" />
+<a href="https://github.com/toddeTV/quick-conf/graphs/contributors">
+<img src="https://contrib.rocks/image?repo=toddeTV/quick-conf" />
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)